girder/api/v1/user.py

```diff
@@ -42,7 +42,8 @@ def __init__(self):
         self.route('GET', ('me',), self.getMe)
         self.route('GET', ('authentication',), self.login)
         self.route('GET', (':id',), self.getUser)
-        self.route('GET', (':id', 'details'), self.getUserDetails)
+        self.route('GET', (':id', 'count'), self.getUserDetails)
```

Comment:
I missed this before -- there should not be an `:id` token in the path, since this does not refer to a specific user but the whole collection of users in the system.